libs/net/dal/Services/ContentService.cs

```diff
@@ -314,9 +314,14 @@ public async Task<IPaged<API.Areas.Services.Models.Content.ContentModel>> FindWi
 
         var response = await _client.SearchAsync<API.Areas.Services.Models.Content.ContentModel>(s =>
         {
+            
+            if(filter.Status == ContentStatus.Publish)
```

Comment:
Hmm interesting solution.  I kinda like it because it's simple.  But I don't think it's wise.  Let me think about this tonight.  I think there are some simple ways we can do this and control the behaviour in a more intuitive way.

Reply:
Add an argument to the `ContentService.FindWithElasticsearchAsync(string index, ContentFilter filter)`.  Then each place it is used it will specify the index with the configuration option `ElasticOptions.UnpublishedIndex` or `ElasticOptions.PublishedIndex`